scripts/install_python_toolchain.sh

```diff
@@ -35,24 +34,32 @@ function linux_patch_sigfpe_handler {
   fi
 }
 
-$PIP install --upgrade "pip>=8.1"
+$PIP install --upgrade "pip"
```

Comment:
Is there a reason you removed the `>=8.1`?

Reply:
I think pip 8.1 is really old now. We don't have to hardcode it. The current pip is already 20.0.2. `>=8.1` is redundant.

Reply:
ok, you have the `--upgrade` so that should work.